script.js

```diff
@@ -169,9 +149,26 @@ const checkIfMatch = (event) => {
   }
 };
 
+const checkIfWon = () => {
```

Comment:
missing function docstring

Reply:
Also added a docstring for `checkIfMatch()`